server/mdm/apple/commander.go

```diff
@@ -0,0 +1,170 @@
+package apple_mdm
```

Comment:
This was moved here from the `service` package, so that it can be used also in the datastore for tests (and it better fits there conceptually, I think).

Reply:
makes a lot of sense, this will conflict with https://github.com/fleetdm/fleet/pull/10971, but I think it's better if we get this PR merged first.